copy/copy.go

```diff
@@ -18,6 +17,7 @@ import (
 	"github.com/containers/image/signature"
 	"github.com/containers/image/transports"
 	"github.com/containers/image/types"
+	gzip "github.com/klauspost/pgzip"
```

Comment:
(Non-blocking nit: I’d slightly prefer using the explicit `pgzip` name throughout the code, to make it obvious that this is not the standard-library implementation.)

Reply:
fixed